Campo IF tecnico 819

MariaNavarro90 opened the pull request:
# Información de la Tarea
Vincular el #819 

### **Resumen de la Solución:** 
-Se agrego un nuevo boton para cargar el numero del IF del informe tecnico. 

### **Información Adicional:**
-

# Descripción de los Cambios

### **Cambios:**
-

# Cómo Testear los Cambios

### **Pruebas Automáticas:**
-

### **Prubeas Manuales:**
-

# Capturas de Pantalla
<img width="1085" height="925" alt="image" src="https://github.com/user-attachments/assets/c604b19a-dae1-4e87-ac54-4e8c182fb1aa" />

MariaNavarro90 committed:
Agregado IFInformeTecnicoForm para que sea obligatorio el campo 'numero_if_tecnico'

MariaNavarro90 committed:
Migracion

MariaNavarro90 committed:
Añadi el campo 'numero_if_tecnico' al modelo de Admisión para el seguimiento de informes técnicos.

MariaNavarro90 committed:
Implementado el manejo del formulario IFInformeTecnico en AdmisionService para guardar el número IF del Informe Técnico.

MariaNavarro90 committed:
Agregado manejo del modal para IF Informe Técnico y visualización del número IF en el formulario de admisión.

MariaNavarro90 committed:
Merge branch 'development' of https://github.com/dsocial118/SISOC-Backoffice into CampoIFtecnico_819

MariaNavarro90 committed:
djlint

MariaNavarro90 committed:
Black

MariaNavarro90 committed:
Merge branch 'development' of https://github.com/dsocial118/SISOC-Backoffice into CampoIFtecnico_819

dsocial118 commented on admisiones/models/admisiones.py:
```diff
@@ -132,6 +132,9 @@ class Admision(models.Model):
         verbose_name="Tipo de observación",
     )
     informe_sga = models.BooleanField(default=False, verbose_name="Estados Informe SGA")
+    numero_if_tecnico = models.CharField(
```
Bloqueante: Esto se agrega, pero no tiene null=True ni default. Por ende van a fallar las admisiones ya creadas.
Sugerencia: Expandir el max_length a 255

MariaNavarro90 committed:
Merge branch 'development' of https://github.com/dsocial118/SISOC-Backoffice into CampoIFtecnico_819

MariaNavarro90 committed:
Comentario de PR:
Bloqueante: Esto se agrega, pero no tiene null=True ni default. Por ende van a fallar las admisiones ya creadas.
Sugerencia: Expandir el max_length a 255

Se agrego default:"" y se amplio a max 255

MariaNavarro90 committed:
null true

MariaNavarro90 committed:
migracion

juanikitro committed:
Merge branch 'development' into CampoIFtecnico_819
